src/Object/Partial.ts

```diff
@@ -0,0 +1,12 @@
+import {OptionalPart} from './Optional'
+import {Depth} from './_Internal'
+
+/**
+Make a type's fields optional. Choose to do it deeply or not.
+@param O the type to make its fields optional
+@param depth (?=`'flat'`) to do it deeply
+@returns [[Object]]
+@example Partial<MyType>
```

Comment:
Examples should follow this form:

```
@example
.```ts
import {...} from 'ts-toolbelt'

// this must be a self contained example
// that can we either copy pasted or
// that will be runnable on the playground
// (for when the new doc website arrives)
.```
```

Reply:
Don't leave the dot in front of .```